Engines/Wine/QuickScript/Steam Script/script.js

```diff
@@ -116,11 +118,20 @@ SteamScript.prototype.go = function () {
         .wizard(setupWizard)
         .prefix(this._name, this._wineDistribution, this._wineArchitecture, this._wineVersion)
         .luna();
+    wine.corefonts();
 
     // Steam must be started once such that config.vdf is created (see fixCertificateIssue())
     setupWizard.wait(tr("Please follow the steps of the Steam setup. Then, wait until Steam is updated, log in and finally close Steam completely so the installation of \"{0}\" can continue.", this._name));
     wine.run(tempFile, [], null, false, true);
 
+    // Set windows environment for executable that needs it
+    wine.setOsForApplication().set("steam.exe", "winxp").do();
+    wine.setOsForApplication().set("steamwebhelper.exe", "winxp").do();
+
+    // Fix for Uplay games that are executed on steam
+    wine.setOsForApplication().set("upc.exe", "winvista").do();
```

Comment:
@plata Should we set it here ? It is a simple way to do IMHO ^^.

Reply:
@ImperatorS79 how else you want to do it assuming that we are not using quick uplay nor uplay script

Reply:
We could chose it is not the responsibility of SteamScript to handle it and for any Steamapps using Uplay you had to set it in the .preInstall. But I think your solution is clean.

Reply:
@ImperatorS79 I'm for sanitization your suggestion is better, proceed.

Reply:
It shouldn't hurt. I'm not sure about the "steamwebhelper.exe". Does this really work still?

Reply:
@plata 
> I'm not sure about the "steamwebhelper.exe". Does this really work still?

Works on my system (tm)